parser-typechecker/src/Unison/Codebase/Editor/Input.hs

```diff
@@ -109,10 +109,10 @@ data Input
   | ExecuteI String
   | TestI Bool Bool -- TestI showSuccesses showFailures
   -- metadata
-  -- link from to
-  | LinkI [Path.HQSplit'] Path.HQSplit'
-  -- unlink from to
-  | UnlinkI [Path.HQSplit'] Path.HQSplit'
+  -- `link metadata definitions` (adds metadata to all of `definitions`)
+  | LinkI HQ.HashQualified [Path.HQSplit'] 
+  -- `unlink metadata definitions` (removes metadata from all of `definitions`)
+  | UnlinkI HQ.HashQualified [Path.HQSplit']
```

Comment:
Some cleanup: switched `LinkI` and `UnlinkI` to take args in the same order as their input pattern, it's `link somedoc def1 def2`.